draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -195,8 +195,8 @@ are mapped to the terminology defined in this document, using cbor and cose.
 RFC9162_SHA256 requires the following:
 
 - TBD_1 (verifiable-data-structure): 1, the integer representing the RFC9162_SHA256 verifiable data structure algorithm.
-- TBD_2 (inclusion-proof): a bstr representing the RFC9162_SHA256 inclusion proof
-- TBD_3 (consistency-proof): a bstr representing the RFC9162_SHA256 consistency proof
+- TBD_2 (inclusion-proof): a bstr or list of bstr representing RFC9162_SHA256 inclusion proofs
```

Comment:
Wouldn't it be better/simpler for this to ALWAYS be a list and sometimes only have 1 element?